File: 백준/10818 최소, 최대.cpp
#include <iostream>
using namespace std;
int main(){
  int n;
  cin>>n;
  int min=1000000;
  int max=-1000000;
  int arr[n];
  for(int i=0; i<n; i++){
    cin>>arr[i];
    if(min>arr[i]) min=arr[i];
    if(max<arr[i]) max=arr[i];
  }
  cout<<min<<" "<<max<<"\n";
}

File: js-baekjoon/bronzeIII/10818최소,최대.js
const fs = require("fs");
const file = process.platform === "linux" ? "/dev/stdin" : "./example.txt";
const input = fs.readFileSync(file).toString().trim().split("\n");
const [t] = input.shift().split(" ").map(Number);
const numbers = input[0].split(" ").map(Number);
console.log(Math.min(...numbers), Math.max(...numbers));
